### PR TITLE
Format and correct README.md syntax

### PR DESCRIPTION
**Pull request by syscl**
as tiltle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ git submodule update --init --recursive
 
 #### Install Dependencies
 
-On MacOS
+On macOS
```
Comment (wenleix) [inline README.md:53]: The subtitle is intended to be `MacOS`, following PyTorch readme: https://github.com/pytorch/pytorch#install-dependencies

Comment (wenleix) [reply README.md:53]: hmm.. seeing `macOS` as well https://github.com/pytorch/pytorch#install-pytorch

Comment (syscl) [reply README.md:53]: Thanks @wenleix for reviewing, yes, the official name of the macOS has been `Mac OS X` then to `OS X` and now `macOS`, I saw the inconsistency in the readme thus change it .

**Review by wenleix — APPROVED** [review]
Thanks for the contribution! You need to import to phab in order to land it.

**Comment by syscl** [comment]
Thanks @wenleix , imported

**Pull request closed without merging**